src/diffusers/pipelines/stable_diffusion_xl/pipeline_stable_diffusion_xl.py

```diff
@@ -582,6 +607,7 @@ def __call__(
         original_size: Optional[Tuple[int, int]] = None,
         crops_coords_top_left: Tuple[int, int] = (0, 0),
         target_size: Optional[Tuple[int, int]] = None,
+        negative_conditions: Dict[str, Tuple[int, int]] = None,
```

Comment:
Even though we already have lots of function input arguments here, I would tend towards doing the following:
```suggestion
        negative_original_size: Optional[Tuple[int, int]] = None,
        negative_crops_coords_top_left: Tuple[int, int] = (0, 0),
        negative_target_size: Optional[Tuple[int, int]] = None,
```

Reply:
I think this is more consistent